.github/workflows/osv-scanner.yaml

```diff
@@ -0,0 +1,145 @@
+# Copyright 2025 Google LLC
```

Comment:
Can you please move this to a separate PR?

Also, let us try to keep this as simple and as close as possible to the example workflows at https://github.com/google/osv-scanner.  Running this on schedule should be sufficient; again, we do not need to create noise in CI-checks for our contributors.  (we have no large scale continuous deployment of Cirq so it is not that critical to catch vulnerabilities on the spot.  Also the only kind of PRs that can introduce them are changes Python dependencies or GHA workflows)